gradle/dependencies.gradle

```diff
@@ -16,9 +16,10 @@ dependencies {
   api('io.github.bonigarcia:webdrivermanager:3.7.1')
 
   implementation('com.browserup:browserup-proxy-core:2.0.1')
-  implementation('io.netty:netty-codec:4.1.42.Final')
+  implementation('io.netty:netty-all:4.1.44.Final')
```

Comment:
Do we really need `netty-all` dependency?
Is `netty-all` contains `netty-codec` dependency?

Reply:
BrowserMobProxy always depended on `io.netty:netty-all` (but just older version). 
But yes, it seems that we don't need `io.netty:netty-codec` because `netty-all` already contains everything. I have removed it.